packages/create-redwood-rsc-app/src/telemetry.ts

```diff
@@ -0,0 +1,90 @@
+import ci from 'ci-info'
+import fetch from 'node-fetch'
+
+import { Config } from './config.js'
+
+const TELEMETRY_URL = process.env.REDWOOD_REDIRECT_TELEMETRY ?? 'https://telemetry.redwoodjs.com/api/v1/telemetry'
+
+// Note: The fields and their names are constrained by the telemetry API
+interface TelemetryPayload {
+  ci: boolean,
+  command: string,
+  complexity: string
+  duration: number,
+  error?: string
+  experiments?: string[],
+  redwoodCi: boolean
+  system: string,
+  type: 'command',
+}
+
+function buildPayload(config: Config | null, duration: number): TelemetryPayload {
+  const command = ['create', 'redwood-rsc-app']
+  if(process.argv.includes('--no-check-latest')){
+    command.push('--no-check-latest')
+  }
+
+  // We don't have a field for the template, so we're using/abusing the experiments field
+  const experiments: string[] = []
+  if(config?.template){
```

Comment:
Project config for crwrsca isn't ideal yet. Can you please try to manually run prettier on this file and see if it changes anything?

Reply:
A couple of files changed in doing so. Did I screw it up even more or? 😆

Reply:
No, all good 👍